pallets/trustless-bridge/bridge-inbound-channel/src/test.rs

```diff
@@ -625,13 +658,13 @@ fn test_message_dispatched() {
 }
 
 #[test]
-fn test_message_dispatched_refund() {
+fn test_batch_dispatched_refund() {
```

Comment:
It would be nice to have test with several dispatched messages

Reply:
In one batch.